mirage/index.js

```diff
@@ -2,8 +2,14 @@
 
 let start = () => {};
 
-if (process.env.NODE_ENV !== 'production') {
-  start = require('./start').default;
-}
+// currently we run mirage in production
+// when we switch over to the initial api protect this
+// require with an if statement so that webpack won't
+// attempt to bundle mirage.
```

Comment:
and this sentence goes on and on and on and ooooooon; don't stop; punctuatin'; dun dun dun; hold on to this feeling